accessibility/common-remediations.md

```diff
@@ -103,13 +133,29 @@ We _don’t_ do this:
 <a onclick=”myFunc();”>Visit this location</a>
 ```
 
+### No advance warning when opening new window
+
+#### What's the problem?
+
+Links open in new window without warning the user that this will happen.
+
+#### Who's affected by the problem?
+
+* Screen reader users
+* Low vision users who magnify the entire screen
+* Users with cognitive challenges
+
+#### Why's it a problem?
 
-### No advance warning when opening new window 
+Having a window open unexpectedly when you click on a link can be disorientating to users who can only see a portion of the screen, who can't see the screen at all, or who can become easily distracted by unexpected changes to their browsing context. 
 
-Avoid opening links in new windows
-This can be disorientating to screen reader users or users with cognitive disabilities.
+#### How do I fix it? 
 
-If you must do it, warn the user before they click on the link that it'll open in a new window. You can use text like "opens in a new window" or a visual icon. If you choose to use an icon, make sure it's accessible to screen reader users.
+Always avoid opening links in new windows wherever possible. 
+
+If you really have to do it, warn the user before they click on the link that it'll open in a new window. You can use text like "opens in a new window" or a visual icon. If you choose to use an icon, make sure it's accessible to screen reader users. 
+
+Note: you MUST use `rel="noopener"` on outbound links that open in a new window. See the [secure markup guide](../security/secure-markup.md/#add-relnoopener-to-outbound-links-in-new-windows) for an explanation. 
```

Comment:
I think that "Please note that you MUST..." would sound better than "Note: you MUST".

Reply:
Normally I'd agree that being polite is better, but this is technical doc that tells people what to do, so social niceties are elided. E.g. WCAG 2.1 has exactly two instances of the word "please", [both in the introduction](https://www.w3.org/TR/WCAG21/), and plenty of "Note" boxes where pleasantries are omitted. [RFC 2119](https://tools.ietf.org/html/rfc2119) (which we reference, as does WCAG 2.1) has no instances of it at all. 

I know we've talked about friendliness in the Playbook before (not you and me, specifically, but as a team), but it does keep coming up. Maybe it's something we need to talk about more widely with the group?

Reply:
🤷‍♂ 

To be fair, I don't mind the "please", I just find the "Note:" bit just weird and rude and unnecessary.

Reply:
Interesting! I use "Note:" all the time for making side notes about things and it's never occurred to me that people might think it's super rude 😬 

How do you feel about dropping it entirely so it says this:

```suggestion
You MUST use `rel="noopener"` on outbound links that open in a new window. See the [secure markup guide](../security/secure-markup.md/#add-relnoopener-to-outbound-links-in-new-windows) for an explanation. 
```

Reply:
Optimus prime 👌